website/docs/Installation/windows-install.md

```diff
@@ -3,69 +3,44 @@ sidebar_position: 4
+![img1](img/windows/homescreen.png)
 
-Once Podman is installed, you will see a toggle button at "Home" window that will allow you to initialize a Podman Machine with default configurations. Simply activate the toggle to proceed.
+**a.To install Podman, click on the `Install` button on the home screen. This will check for all the requirements for installation of Podman Engine.**
 
-![img2](img/initialize.png)
+![img2](img/windows/prereq-wsl2.png)
 
-If this is your first time initializing Podman Machine after installation and you do not have WSLv2 installed, Podman Desktop will prompt you to do so (as shown in image below). Click 'OK' and wait for the process to complete. 
+**b. If you do not have WSL2 installed, follow the instructions [here](https://learn.microsoft.com/en-us/windows/wsl/install-manual) to install the same.**
 
-![img3](img/wsl_before_reboot_1.png)
+**c. Once all the requirements are met, Podman Desktop will ask you to allow installation of Podman (Engine). Click on `Yes` to continue.**
 
-Your device will reboot during the process. You will be again prompted to proceed.
+![img3](img/windows/podman-install.png)
 
-![img4](img/wsl_before_reboot_2.png)
+**4. Once installation is completed, you are ready to use Podman Desktop.**
 
- After reboot, the remaining installation process will be performed. Once WSLv2 is installed, you will find that the Podman Machine is initialized.
+After the installation of Podman Engine, your home screen would like the one in the image below.
 
-![img5](img/wsl_after_reboot.png)
+![img4](img/windows/podman-desktop-ready.png)
 
-After initializing a Podman Machine, you should see a toggle to Run Podman. This will start the Podman Machine upon activation.
-
-![img6](img/starting.png)
-
-### b. Custom Configurations
-
-In order to initialize a Podman Machine with custom configurations, go to "Preferences" on the menu present in the left-side of the application. Under Resources, you will find Podman. Clicking on it shall load the configuration settings for the machine. Enter the values that deem fit for your purpose and click on the "Create" button.
-
-![img7](img/create.png)
-
-Once the machine is created, you can click on the Start button in the Machine Settings to start the Machine.
-
-![img8](img/machine.png)
-
-### c. Command Line
-
-Using the following two commands in the command line, you can initialize and start a Podman Machine the classic way!
-
-To initialize the machine, the command is
-
-```sh
-podman machine init
-```
-
-After which, you can start the machine with the command
-
-```sh
-podman machine start
-```
-
-**Well that's just it. You shall now be all set to use Podman Desktop on Windows!**
+If the screen says `Podman is installed but not ready`, it means that installation of Podman Engine has been completed but a Podman Machine has not been initialized. 
 
+You can learn more about working with Podman Desktop in our [Getting Started guide](/docs/getting-started/getting-started).
```

Comment:
Please add a section for that.
We need to be consistent with the structure between all "installation guides" 
On mac, we did have a section title "Next Steps" : https://podman-desktop.io/docs/Installation/macos-install#next-steps

Reply:
done